app/Http/Controllers/Auth/LoginController.php

```diff
@@ -225,6 +226,11 @@ private function externalRealopsLogin($cid, $fname, $lname, $email) {
 
     private function completeRealopsLogin($realops_pilot) {
         auth()->guard('realops')->login($realops_pilot);
-        return redirect('/realops');
+
+        if(session('realops_redirect')) {
+            return redirect('/realops');
+        }
```

Comment:
Should we clear the realops redirect session variable after this? If a user then tries to login as a controller after this, I don't think it would let them

Reply:
I guess it wouldn't hurt either way to clear the session variable out. I'll do that

Reply:
I need to put this in a QA environment and just make sure it doesn't cause an issue. I can just see this causing an issue or bugs in the future even if it doesn't cause something now

Reply:
Added directive to clear the session variable after it is used